doc/source/conf.py

```diff
@@ -127,7 +129,14 @@
     "pytwin": ("https://twin.docs.pyansys.com/version/stable/", None),
 }
 
-suppress_warnings = ["label.*", "design.fa-build", "config.cache"]
+# ref.ref warning needs to be re-added when merging to main
+suppress_warnings = [
+    "label.*",
+    "design.fa-build",
+    "config.cache",
+    "ref.ref",
+    "misc.highlighting_failure",
+]
```

Comment:
All of these warnings need to be suppressed?

Reply:
Sure, we can suppress them as warnings are ignored in the build process!

Reply:
So those warnings, need to be suppressed?

Reply:
Yes, I am deleting them!